modules/serverless-rn-1-7-0.adoc

```diff
@@ -1,52 +0,0 @@
-// Module included in the following assemblies:
```

Comment:
Is there any rule about when we remove the older release notes? Do you want to keep one version back? Thanks

Reply:
I just remove them if they're not supported. We kept 1.10.0 here but removed anything older since they're not supported for 4.6.